### PR TITLE
fix(kit): `TextArea` add ability to override minHeight when using maxLength

### LINKED ISSUE #2872: 🐞 - TextArea not working with custom size and tuiTextfieldMaxLength attribute
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/angular-uae7bc?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.less,src%2Fapp%2Fapp.module.ts

### Is this issue blocking you?

Blocking

### Description

When you try to set a custom height for the textarea and add a max length validation with tuiTextfieldMaxLength, the textarea expands despite the min-length property

### Angular version

14.1.0

### Taiga UI version

2.58.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #2872

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/kit/components/text-area/text-area.component.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ export const DEFAULT_ROWS = 20;
 export const LINE_HEIGHT_M = 20;
 export const LINE_HEIGHT_L = 24;
 
+const DEFAULT_HEIGHT_REM = 6.75;
+const DEFAULT_HEIGHT_COUNTER_REM = 7.875;
+
```
Comment (splincode) [inline projects/kit/components/text-area/text-area.component.ts:47]: should this code be moved to DI options?

**Review by splincode — APPROVED** [review]

File: projects/kit/components/text-area/text-area.component.ts (hunk 1 above)
Comment (waterplea) [reply projects/kit/components/text-area/text-area.component.ts:47]: This is not configurable, it's just for calculations without magic numbers

File: projects/kit/components/text-area/text-area.style.less [hunk 2]
```diff
@@ -19,15 +16,6 @@
 
     &[data-size='m'] {
         font: var(--tui-font-text-s);
-        min-height: (@height - 1.25rem);
```
Comment (waterplea) [inline projects/kit/components/text-area/text-area.style.less:22]: You could still keep it in LESS with no JS, just update value of the variable, i.e.:
```css
&[data-size='m'] {
  font: var(--tui-font-text-s);

  --tui-textarea-height: (@height - 1.25rem);
}
```

Comment (vladimirpotekhin) [reply projects/kit/components/text-area/text-area.style.less:22]: cool, fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**